yat-lib/src/main/java/yat/android/lib/YatIntegration.kt

```diff
@@ -73,25 +74,43 @@ class YatIntegration {
     }
 
     sealed class Environment(val yatAPIBaseURL: String, val yatWebAppBaseURL: String) {
-        object SandBox : Environment("https://a.yat.fyi/", "https://yat.fyi/")
-
-        object Production : Environment("https://a.y.at/", "https://y.at/")
+        data object SandBox : Environment("https://a.yat.fyi/", "https://yat.fyi/")
```

Comment:
This environment is for internal use only. It should't be a part of the framework.